website/settings/celeryconfig.py

```diff
@@ -0,0 +1,34 @@
+from celery.schedules import crontab
+
+
+##### Celery #####
+## Default RabbitMQ broker
+BROKER_URL = 'amqp://'
+
+# Default RabbitMQ backend
+CELERY_RESULT_BACKEND = 'amqp://'
+
+# CELERYBEAT_SCHEDULE_FILENAME = 'celeryconfig.py'
+# Setting up a scheduler
+CELERYBEAT_SCHEDULE = {
+    '5-minute-emails': {
+        'task': 'notify.send_users_email',
+        'schedule': crontab(minute='*/5'),
```

Comment:
The proposal specifies a buffer period of "60 seconds"--why was this changed to 5 minutes?

https://docs.google.com/document/d/1DecEEGgi47wW1ye0d_9e324sy7CzCJY_wWdEcFYu8gc/edit